modules/otel-collector-config-options.adoc

```diff
@@ -85,7 +85,7 @@ spec:
 
 |extensions:
 |Optional components for tasks that do not involve processing telemetry data.
-|`bearertokenauth`, `oauth2client`, `jaegerremotesamplin`, `pprof`, `health_check`, `memory_ballast`, `zpages`
+|`bearertokenauth`, `oauth2client`, `jaeger???remotesamplin`, `pprof`, `health_check`, `memory_ballast`, `zpages`
```

Comment:
keep, there is a typo ` jaegerremotesampling`

Reply:
TY. Fixed.